File: Semestres/Edteam/selectionSort.js
function selection_sort(arreglo){

	arreglo_ord = arreglo.slice();

	for(i=0; i<arreglo.length-1;i++){
		min = arreglo_ord[i];
		pos_min = i;
		for(j=i+1;j<arreglo_ord.length;j++){
			if(arreglo_ord[j]<min){
				min = arreglo_ord[j];
				pos_min = j;
			}
		}

		if(i!=pos_min){
			aux =arreglo_ord[i];
			arreglo_ord[i] = arreglo_ord[pos_min];
			arreglo_ord[pos_min] = aux

		}

	}

	return arreglo_ord;
}

console.log(selection_sort([10,2,3,-1,11,2,3,4,5,6,7,8,9,-2,4]))

File: Semestres/Edteam/selectionSort.py
def selection_sort(arreglo):
    arreglo_ordenado = arreglo.copy()

    for i in range(0,len(arreglo_ordenado)):
        minimum = arreglo_ordenado[i]
        pos_min = i
        for j in range(i+1,len(arreglo_ordenado)):
            if arreglo_ordenado[j]<minimum:
                minimum = arreglo_ordenado[j]
                pos_min = j
        if pos_min != i:
            arreglo_ordenado[i],arreglo_ordenado[pos_min] =arreglo_ordenado[pos_min],arreglo_ordenado[i]
                
    return arreglo_ordenado


